client/src/Components/Box/Box.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { BoxComponent, ImgContainer, ContentContainer, BookParagraph, BoxBookReview, BoxReadMore, BookImage } from './BoxStyles';
+
+export type BoxComponentProps = {
+  title: string;
+  author: string;
+  rate: number;
+  review: string;
+  image: string;
+}
+
+export const Box = ({ title, author, rate, review, image }: any) => {
```

Comment:
Zamieńmy ten typ any

Reply:
Dobra, długo nie wiedziałem jak to ugryźć , Klara właśnie mi pomogła... gotowe